katas/content/distinguishing_states/a_b/index.md

```diff
@@ -0,0 +1,6 @@
+**Inputs:**
+
+1. Angle $\alpha$, in radians, represented as a `Double`.
+2. A qubit which is guaranteed to be either in $\ket{A}$ or the $\ket{B}$ state, where $\ket{A} = \cos \alpha \ket{0} + \sin \alpha \ket{1}$ and $\ket{B} = - \sin \alpha \ket{0} + \cos \alpha \ket{1}$.
```

Comment:
Let's make the states A and B a bit easier to read and compare next to each other
```suggestion
2. A qubit which is guaranteed to be either in $\ket{A}$ or the $\ket{B}$ state, where 
$$\ket{A} = \cos \alpha \ket{0} + \sin \alpha \ket{1}$$ 
$$\ket{B} = - \sin \alpha \ket{0} + \cos \alpha \ket{1}$$
```